Client/Frontend/Widgets/ToolbarButtons.swift

```diff
@@ -0,0 +1,185 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+enum ToolbarButtonTag: Int {
+    case Back
+    case Forward
+    case Reload
+    case Share
+    case Bookmarked
+}
+
+// MARK: - Browser Toolbar Buttons
+extension UIButton {
```

Comment:
don't know if I like this being an extension on UIButton as a base class seeing as it only affects UIButton's that are on a toolbar (same with the toolbar specific global extsions on UIImage below). Wondering if a helper class rather than an extension might be a nicer way to go here, or create a ToolbarButton subclass and then add the extensions to that (or when UIButton is of that subtype).